src/components/StakingVaults/Cells.tsx

```diff
@@ -99,9 +101,9 @@ export const AssetCell = ({
       <HStack flex={1}>
         <SkeletonCircle isLoaded={!!asset} mr={2} width='auto' height='auto'>
           {icons && icons.length > 1 ? (
-            <PairIcons icons={icons} iconSize='sm' bg='none' />
+            <PairIcons icons={icons} iconSize='sm' bg='none' showFirst={true} />
```

Comment:
super nitpick but possibly spread `pairProps` here for reuse, though happy with the declarativeness of this as well

Reply:
Done in [a3ef322](https://github.com/shapeshift/web/pull/7488/commits/a3ef322255b2188c79f85b92961b44cf7c5a651e)